gearcmd/worker.go

```diff
@@ -302,7 +302,14 @@ func stopProcess(p *os.Process, gracePeriod time.Duration) error {
 		}
 		time.Sleep(gracePeriod)
 	}
-	if err := syscall.Kill(p.Pid, syscall.SIGKILL); err != nil {
+	// kill entire group of process spawned by our cmd.Process
+	pgid, err := syscall.Getpgid(p.Pid)
+	if err != nil {
+		return fmt.Errorf("unable to get pgid, error: %s", err)
+	}
+	lg.InfoD("killing-pgid", logger.M{"pid": p.Pid})
+	// minus sign required to kill PGIDs
```

Comment:
Can we flesh this out a bit more?
I think a link to https://linux.die.net/man/2/kill would be useful